### ISSUE TITLE
Failed to parse function with trailing comma

### ISSUE THREAD

**Issue by lambdalisue** [yyx990803/buble#6]
I'm not sure but it seems the following issue is related to my issue on Vue.

## tl;dr

This fork cannot parse the following code but similar code is written in my Vue template.

```
(d, f) =>
  updateNotificationSetting(
    notificationSetting.notificationSettingId,
    d,
  ).finally(f);
```

Btw, the original one can because of https://github.com/Rich-Harris/buble/pull/54

## Detail

I'm using `@vue/cli-plugin-eslint, eslint-config-prettier, and prettier` and after I updates those today, `vue-cli-service lint` conver the following HTML

```
<notification-setting-viewer
 v-else
 :key="`viewer-${notificationSetting.notificationSettingId}`"
 :notification-setting="notificationSetting"
 @delete:notificationSetting="(f) => deleteNotificationSetting(notificationSetting.notificationSettingId).finally(f)"
 @update:notificationSettingDisabled="(b, f) => updateNotificationSettingDisabled(notificationSetting.notificationSettingId, b).finally(f)"
 @update:mode="(m, f) => updateMode(notificationSetting.notificationSettingId, m).finally(f)"/>
```

To

```
<notification-setting-viewer
  v-else
  :key="`viewer-${ns.notificationSettingId}`"
  :notification-setting="ns"
  @delete:notificationSetting="
    f =>
      deleteNotificationSetting(ns.notificationSettingId).finally(f)
  "
  @update:notificationSettingDisabled="
    (b, f) =>
      updateNotificationSettingDisabled(
        ns.notificationSettingId,
        b,
      ).finally(f)
  "
  @update:mode="
    (m, f) => updateMode(ns.notificationSettingId, m).finally(f)
  "
/>
```

With previous HTML, `vue-cli-service build` produce a production code as ES5 compatible but with the new HTML, the production code is not ES5 compatible like

```
   notificationSetting.notificationSettingId,
  d,
).finally(f); },"update:mode":function (m, f) { return _vm.updateMode(
  notificationSetting.notificationSettingId,
  m,
).finally(f); },"test:mail":_vm.sendTestMail}}):_c('notification-setting-viewer',{key:("viewer-" + (notificationSetting.notificationSettingId)),attrs:{"notification-setting":notificationSetting},on:{"delete:notificationSetting":function (f) { return _vm.deleteNotificationSetting(
  notificationSetting.notificationSettingId,
).finally(f); },"update:notificationSettingDisabled":function (b, f) { return _vm.updateNotificationSettingDisabled(
  notificationSetting.notificationSettingId,
  b,
).finally(f); },"update:mode":function (m, f) { return _vm.updateMode(
  notificationSetting.notificationSettingId,
  m,
).finally(f); }}})]})],2)],1),(_vm.loading)?_c('float-loader'):_vm._e()],1)}
```

Note that somehow the trailing comma remains.

And I found that this buble is used in vue-template-compiler and this buble cannot handle trailing comma in function so I wonder if it's related to my issue.

**Comment by yyx990803** [comment]
Now synced against Buble 0.19.3 and trailing function comma is now supported in `vue-template-es2015-compiler@1.9.0`
